### PR TITLE
feat: add Enzyme radspec

### PR DESCRIPTION
**Pull request by luongnt95**

### CODE REVIEW

**Comment by SeanJCasey** [comment]
@luongnt95 This should include all the functions that the Council can call in the current release:

- all `onlyOwner` calls in `Dispatcher`
- all `onlyOwner` calls in `FundDeployer`
- all `onlyFundDeployerOwner` calls in any contract
- all `onlyDispatcherOwner` calls in any contract (e.g., price feeds)

**Comment by luongnt95** [comment]
@SeanJCasey Added!

File: src/data/enzyme/knownFunctions.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  'removeAllowedMakers(address[])': 'Remove allowed makers `$1`',
+  'setSharesTokenSymbol(string)': 'Set shares token symbol `$1`',
+  'removeNominatedOwner()': 'Remove nominated owner',
+  'setCurrentFundDeployer(address)': 'Set current fund deployer `$1`',
+  'setNominatedOwner(address)': 'Set nominated owner `$1`',
+  'setMigrationTimelock(uint256)': 'Set migration time lock `$1`',
+  'setComptrollerLib(address)': 'Set comptroller lib `$1`',
+  'deregisterVaultCalls(address[],bytes4[])': 'De-register vault calls _contract `$1` _selector `$2`',
+  'registerVaultCalls(address[],bytes4[])': 'Register vault calls _contract `$1` _selector `$2`',
+  'addDerivatives(address[],address[])': 'Add derivatives _derivative `$1` _priceFeed `$2`',
```
Comment (buendiadas) [inline src/data/enzyme/knownFunctions.js:18]: This wasn't a thing until today but with the new `SingleUnderlyingDerivativeRegistryMixin` it seems that we are going to have a collision on this function signature:
 -  On `AggregatedDerivativePriceFeed` we have `_derivatives $1, _priceFeeds $2` 
 -  On  `SingleUnderlyingDerivativeRegistryMixin`  we have `_derivatives` `_underlyings`

Comment (buendiadas) [reply src/data/enzyme/knownFunctions.js:18]: cc. @luongnt95 @SeanJCasey

File: src/data/enzyme/knownFunctions.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  'removeDerivatives(address[])': 'Remove derivatives `$1`',
+  'updateDerivatives(address[],address[])': 'Update derivatives _derivative `$1` _priceFeed `$2`',
+  'setEthUsdAggregator(address)': 'Set ETH-USD aggregator `$1`',
+  'addPrimitives(address[],address[],uint256[])': 'Add primitives _primatives `$1` _aggregators `$2` _rateAssets `$3`',
+  'removePrimitives(address[])': 'Remove primitives `$1`',
+  'updatePrimitives(address[],address[])': 'Update primitives _primatives `$1` _aggregators `$2`',
+  'setStaleRateThreshold(uint256)': 'Set staleRateThreshold `$1`',
+  'addPoolTokens(address[])': 'Add uniswap pool tokens to the uniswap pool price feed `$1`',
```
Comment (buendiadas) [inline src/data/enzyme/knownFunctions.js:26]: Here I think @SeanJCasey  wanted to avoid using Uniswap to be able to reuse in a future?

**Review by nivida — APPROVED** [review]

**Pull request merged**